ui/src/utils/workspace.fs.ts

```diff
@@ -0,0 +1,54 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { dirname } from 'path';
+import { FileStat, FileType, Uri, workspace } from 'vscode';
+import { parseError } from '../parseError';
+
+export async function ensureFile(path: string): Promise<void> {
```

Comment:
Based off of https://github.com/jprichardson/node-fs-extra/blob/master/lib/ensure/file.js